internal/services/recoveryservices/recovery_services_vault_resource.go

```diff
@@ -140,6 +141,13 @@ func resourceRecoveryServicesVault() *pluginsdk.Resource {
 				Optional: true,
 				Default:  true,
 			},
+
+			"classic_vmware_replication_enabled": {
+				Type:     pluginsdk.TypeBool,
+				Optional: true,
+				Computed: true,
```

Comment:
Does this need to be computed?

Reply:
updated comment in code, and I'm worried if we set default value for it, it may break some users vault as they might have changed this on Portal.